esp/esdllib/CMakeLists.txt

```diff
@@ -73,7 +73,4 @@ IF (USE_LIBXSLT)
     include_directories (
          ${LIBXML2_INCLUDE_DIR}
     )
-    target_link_libraries ( esdl_svc_engine
```

Comment:
Are the includes still needed, or should the whole if block be removed?

Reply:
The includes are still needed, I'll look at whether it's possible to remove it.